docs/user-guide/QuickStart.md

```diff
@@ -24,7 +22,8 @@ The minimal requirements for fuzzing your API are:
 
 ## **Initial run**
 
-Once you have prepared (1) and (2), run the following command:
+Once you have built RESTler per the steps in the main README, and
+prepared (1) and (2), run the following command:
 
 ```python restler-quick-start.py --apiSpec <path to specification> --restler_drop_dir <path to RESTler drop>```
```

Comment:
This should actually says api_spec_path instead of apiSpec - may want to add the fix to this PR.

Reply:
Nevermind - already fixed in main

Reply:
I need to rebase, that was already fixed by main

---
In reply to: [525434668](https://github.com/microsoft/restler-fuzzer/pull/64#discussion_r525434668) [](ancestors = 525434668)